src/components/datagrid/controls/_data_grid_display.scss

```
@@ -1,3 +1,3 @@
-.euiDataGrid__displayFormRow {
-  width: $euiFormMaxWidth;
+.euiDataGrid__stylePopoverPanel {
```

Can we prefer not using the `style` naming for future-proof-ness? Once we add the row height switcher, this panel is no longer affecting just `gridStyles`, so I think a more generic `display` name makes sense

```suggestion
.euiDataGrid__displayPopoverPanel {
```